tensorflow/core/kernels/batchtospace_op.cc

```diff
@@ -23,7 +23,6 @@ limitations under the License.
 
 #include "tensorflow/core/kernels/spacetobatch_functor.h"
 
-#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
```

Comment:
Actually, I think we keep this include here because Eigen defines some #defines that must be defined prior to other headers (sadness).  Is it okay to put it back where it was?

Reply:
Thanks @vrv. Didn't know that as I was just using `clang-format -I` to reformat the .cc file.

The PR has been updated with `"third_party/eigen3/unsupported/Eigen/CXX11/Tensor"` back.
 Please take a look.

Reply:
Yup, thanks :)